src/modules/Favorites/components/FavoriteTracks.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { View, StyleSheet, FlatList } from 'react-native';
+import { useSelector, useDispatch } from 'react-redux';
+import { Navigation } from 'react-native-navigation';
+
+import { colors } from '../../../constants/colors';
+import { Track } from '../../Artists/components/Track';
+import { Header } from '../../../components/Header';
+import { selectFavoriteAlbums } from '../../Favorites/selectors';
+import { removeFavoriteAlbum } from '../actions';
+
+export const FavoriteTracks = ({ componentId, albumId, collectionName }) => {
+  const favoriteAlbum = useSelector(selectFavoriteAlbums(albumId));
```

Comment:
You could create selector for `favoriteTracks` (`useSelector(selectFavoriteTracksByAlbumId(albumId))`)